src/org/labkey/test/tests/AdvancedImportOptionsTest.java

```diff
@@ -109,6 +111,9 @@ public void doCleanup(boolean afterTest) throws TestTimeoutException
         if (_studyDesignPreviouslyEnabled != null)
             OptionalFeatureHelper.setOptionalFeature(createDefaultConnection(), "studyDesignFlag", _studyDesignPreviouslyEnabled);
 
+        if (_advancedImportOptionsEnabled != null)
+            OptionalFeatureHelper.setOptionalFeature(createDefaultConnection(), "advancedImportFlag", false);
```

Comment:
Don't we want to use the initial state of the flag that was stashed away in `_advancedImportOptionsEnabled`?

Reply:
ProgressReportTest could be used as an example:
```
    @Override
    protected void doCreateSteps()
    {
        _studyDesignPreviouslyEnabled = OptionalFeatureHelper.enableOptionalFeature(createDefaultConnection(), "studyDesignFlag");

```
...
```
    @Override
    protected void doCleanup(boolean afterTest)
    {
        if (_studyDesignPreviouslyEnabled != null)
            OptionalFeatureHelper.setOptionalFeature(createDefaultConnection(), "studyDesignFlag", _studyDesignPreviouslyEnabled);
    }

```

Reply:
Thank you, both. Now updated.